canonical-web-and-design/jaas.ai
# Remove homepage to use discourse topic

## Done

Remove homepage and use permanent topic: https://discourse.jujucharms.com/t/juju-documentation-index/1087

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8029/docs
- Should see the content of the topic
